README.md

```diff
@@ -15,6 +15,13 @@ Check out our website [here](https://www.ladybug.dev/)!
 
 Our podcast website is built using [Gatsby](https://gatsbyjs.org) and is fully open source! Find a bug? Think something could be better presented? Open a PR or issue and let us know!
 
+## How to use
+
+1. Fork and clone
+1. Run 'yarn' to install requirements
+1. Run 'yarn start' or 'gatsby develop' to run server
+1. View locally: http://localhost:8000
```

Comment:
You can also add a note that you can go to `http://localhost:8000/__graphql` to get to the GraphQL playground!

Also please update the numbers to increment - I only  see 1's :)